syntax = "proto2";

message ResponseMessage {
    optional uint32 lidar1 = 1;
    optional uint32 lidar2 = 2;
}

message RequestMessage {
    optional uint32 fibRequest = 1;
}
